dist/index.js

```diff
@@ -45,11 +47,13 @@ var ReactScrollPagination = _react2.default.createClass({
   isolate: {
     onePageHeight: null,
     timeoutFuncHandler: null,
+    excludeTopMargin: null,
```

Comment:
@malikid 
   Thanks so much for your pr, one question, is the prop `excludeTopMargin` works same as the prop `excludeHeight`, if so, maybe you can comment hte `excludeHeight` as deprecated.